internal/provisioner/bundledeployment/bundledeployment.go

```diff
@@ -468,3 +498,51 @@ func isResourceNotFoundErr(err error) bool {
 	//   the use of string comparisons for now.
 	return strings.Contains(err.Error(), "no matches for kind")
 }
+
+// SetupWithManager sets up the controller with the Manager.
+func (p *bundledeploymentProvisioner) SetupWithManager(mgr ctrl.Manager) error {
+	controller, err := ctrl.NewControllerManagedBy(mgr).
+		For(&rukpakv1alpha1.BundleDeployment{}, builder.WithPredicates(
+			util.BundleDeploymentProvisionerFilter(plain.ProvisionerID)),
+		).
```

Comment:
I haven't had a chance to look over the new directory structure since #447 landed: we're specifying the plain.ProvisionerID here, and filtering BundleDeployments that don't match that unique provisioner class ID from the cache. Is that the right read here? If so, I think we'll need to solve this in an agnostic way s.t. our core provisioners can leverage this functionality without inter-depending on the plain provisioner libraries.

Reply:
Oh, it's my mistake.  This function is not used any more since #447 changes.   I'll take this out.